lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1091,8 +1091,10 @@ ONNXModelLoader::loadConstantOfShape(const ONNX_NAMESPACE::NodeProto &op,
                       "Input element type must be Int64ITy.");
     // Convert 1D tensor of int64_t into llvm::ArrayRef<size_t>.
     auto TH = in->getPayload().getHandle<int64_t>();
-    llvm::ArrayRef<size_t> outputDims = {(const size_t *)TH.begin(),
-                                         (const size_t *)TH.end()};
+    auto begin = &TH.raw(0);
```

Comment:
can use pointers to begin() and end() here

Reply:
Can't. I changed the iterator to implement random access, but still not able to convert it to ArrayRef. It doesn't have constructor from `{TH.begin(), TH.end()}`